app/pages/lab/workflow-create-dialog.cjsx

```diff
@@ -0,0 +1,58 @@
+React = require 'react'
+ReactDOM = require 'react-dom'
+counterpart = require 'counterpart'
+
+WorkflowCreateDialog = React.createClass
+  getDefaultProps: ->
+    onCancel: ->
+    onSubmit: ->
+    onSuccess: ->
+    projectID: ''
+    workflow: null
+
+  getInitialState: ->
+    busy: false
+    error: null
+
+  handleSubmit: (e) ->
+    e.preventDefault()
+
+    if not @refs.newDisplayName.value
```

Comment:
Use `<input required />` when possible instead of checking for form values manually.